cicd/pipelines/customizing-configurations-in-the-tektonconfig-cr.adoc

```diff
@@ -0,0 +1,56 @@
+:_content-type: ASSEMBLY
+[id="customizing-configurations-in-the-tektonconfig-cr"]
+= Customizing configurations in the TektonConfig custom resource
+include::_attributes/common-attributes.adoc[]
+:context: customizing-configurations-in-the-tektonconfig-cr
+
+toc::[]
+
+In {pipelines-title}, you can customize the following configurations by using the `TektonConfig` custom resource (CR):
+
+* Configuring the {pipelines-title} control plane
+* Changing the default service account
+* Disabling the service monitor
+* Configuring pipeline resolvers
+* Disabling cluster tasks and pipeline templates
+* Disabling the integration of {tekton-hub}
+* Disabling the automatic creation of RBAC resources
+* Pruning of task runs and pipeline runs
+
+[id="prerequisites_customizing-configurations-in-the-tektonconfig-cr"]
+== Prerequisites
+
+* You have installed the {pipelines-title} Operator.
+
+include::modules/op-configuring-pipelines-control-plane.adoc[leveloffset=+1]
+
+include::modules/op-configuration-fields-with-default-values.adoc[leveloffset=+2]
+
+include::modules/op-optional-configuration-fields.adoc[leveloffset=+2]
+
+include::modules/op-changing-default-service-account.adoc[leveloffset=+1]
+
+include::modules/op-disabling-the-service-monitor.adoc[leveloffset=+1]
+
+include::modules/op-configuring-pipeline-resolvers.adoc[leveloffset=+1]
+
+include::modules/op-disabling-cluster-tasks-and-pipeline-templates.adoc[leveloffset=+1]
+
+include::modules/op-disabling-the-integretion-of-tekton-hub.adoc[leveloffset=+1]
+
+include::modules/op-disabling-automatic-creation-of-rbac-resources.adoc[leveloffset=+1]
+
+include::modules/op-automatic-pruning-taskrun-pipelinerun.adoc[leveloffset=+1]
+
+include::modules/op-default-pruner-configuration.adoc[leveloffset=+2]
+
+include::modules/op-annotations-for-automatic-pruning-taskruns-pipelineruns.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+[id="additional-resources_customizing-configurations-in-the-tektonconfig-cr"]
+== Additional resources
```

Comment:
I believe that according to the [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections) we are supposed to avoid putting paragraphs in the additional resources like this, and instead just include the links